app/scripts/metamask-controller.js

```diff
@@ -385,7 +387,7 @@ export default class MetamaskController extends EventEmitter {
             rpcPrefs: {},
             rpcUrl: 'http://localhost:8545',
             ticker: 'ETH',
-            networkConfigurationId: 'networkConfigurationId',
+            id: 'networkConfigurationId',
```

Comment:
These initialNetworkControllerState changes are to accommodate unreleased NetworkController changes that are being brought into the preview build that is used in this branch. They are unrelated to removing networkId, will likely be addressed in a separate PR to extension, and can be ignored for now

Reply:
These NetworkController changes have just been released in https://github.com/MetaMask/core/pull/1697 and should be adopted in extension via separate PR.